pkg/chartutil/create.go

```diff
@@ -238,6 +248,15 @@ spec:
     app.kubernetes.io/name: {{ include "<CHARTNAME>.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
 `
+const defaultServiceAccount = `{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "<CHARTNAME>.serviceAccountName" . }}
+  labels:
+  {{ include "<CHARTNAME>.labels" . | indent 4 }}
```

Comment:
this should probably use `nindent` instead.

Reply:
`nindent` adds a new line at the start. But `{{` is used at the start of this to the prior whitespace is kept including a new line. I think `indent` works but to be consistent with the other files created the two spaces before the `{{` need to be removed.

Reply:
Thanks!